eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml

```diff
@@ -15,7 +15,7 @@ parameters:
   stagedBuild: false
   displayNameArgs: ''
   runInUnloadableContext: false
-  runtimeVariant: ''
+  runtimeVariant: 'interpreter'
```

Comment:
what's the difference between `monointerpreter` and `interpreter`?

Reply:
Should be monointerpreter, will fix.